src/main/kotlin/no/skatteetaten/aurora/boober/feature/fluentbit/FluentbitConfigurator.kt

```diff
@@ -235,10 +235,10 @@ class FluentbitConfigurator {
             |   Match *
             |   Add   host $ {POD_NAME}
             |   Add   environment $ {POD_NAMESPACE}
-            |   Add   version $version
+            |   Add   applicationVersionUnique $version
```

Comment:
Kanskje en liten kommentar om at disse navnene ikke må krasje med noen navn som er i bruk et annet sted?

Er jo ikke helt intuitivt hvorfor det ikke gjelder for environment, host etc...

Reply:
> Er jo ikke helt intuitivt hvorfor det ikke gjelder for environment, host etc...

Nei, det kan faktisk være det er verdt å gjøre det for dem også; "host" og "environment" er jo relativt vanlige navn på variabler som kan tenkes å finnes i json-strukturen.